src/components/Header/index.tsx

```diff
@@ -8,11 +8,13 @@ import { useMediaQuery } from '../../hooks/useMediaQuery';
 interface Props {
   darkModeController?: typeof defaultDarkModeController;
 }
+var isCurrentUrl= window.location.href
```

Comment:
Delete this. We are using Gatsby and have access to <Link> to switch between views